governance.md

```diff
@@ -153,49 +289,135 @@ are enumerated below.
+timezones.
+
+If someone becomes unable or unwilling to continue in their roles, they may retire. If someone doesn't
+fulfill their role for 90 days or violates the code of conduct, they may be removed from the role
+(escalation/vote process TBD). If they wish to resume their role in the future, they may request to return
+to it by asking the current members filling that role.
+
+# Special Interest Group (SIG) Governance
+
+
+[SIGs](README.md#special-interest-groups-sig-and-working-groups) are the subteams of the project.
+SIGs own code and/or documentation of the project, share knowledge, bring new members up to speed, and
+more.
+
+We have a couple dozen SIGs, and contributors in more than a dozen timezones. People want to be able
+to follow what's going on without attending every meeting, which is impossible, especially for
+non-American timezones.
+
+In order to standardize Special Interest Group efforts, maximize transparency, and route contributors
+to the appropriate SIG, SIGs should follow the guidelines stated below:
+
+- Meet regularly by video conference, phone conference, or in person, at least for 30 minutes every month
+  except November and December
+- Keep up-to-date meeting notes, linked from the SIG's page in the community repo
+- Announce meeting agenda before each meeting and post minutes after, on their SIG mailing list
+- Record SIG meeting and make it publicly available
+  - TBD: Some SIG leads have objected to this, and also pointed out that it’s generally less useful
+    than notes (low information density, not searchable, not skimmable)
+- Ensure the SIG's mailing list and slack channel are archived
+- Report activity in the weekly community meeting at least once every 6 weeks
+- Participate in release planning meetings and retrospectives, and burndown meetings, as needed.
+  When the right people aren't present in such meetings, it can put the project at risk, such as
+  by slipping the release.
+- Development of new code  happens in a project-owned github org and repository, with code and tests 
+  explicitly owned and supported by the SIG, including issue triage, PR reviews, test-failure response,
+  bug fixes, etc.  An exception is when the SIG contributes to existing OSS projects not on github (e.g.
+  Apache Spark) or which have their own github org (e.g. Tensorflow) to improve integration with
+  Kubernetes 
+- Use the above forums as the primary means of working, communicating, and collaborating, as opposed to
+  private emails and meetings
+- If used, Google Docs should be made available to everyone on the project. Share them with at least the
+  SIG discussion group and kubernetes-dev, and it is recommended that they be made public. When sharing
+  with kubernetes-dev, do not notify the list, unless it is really an issue that everyone on the project
+  should immediately be made aware of.
+- Represent the SIG for the PM group (either a SIG liaison to the PM group or a PM liaison to the SIG):
```

Comment:
I'd like to remove this until we have some wide agreement on how the PM group *should* operate.  Right now there is no wide agreement there.